.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 17
+      - uses: actions/cache@v1
```

Comment:
I seem to recall that manual caching becomes unnecessary, though maybe that requires a newer setup-java?

https://stackoverflow.com/a/70712541/28465 looks to back me up.

But assuming that this build ends up passing, let's just roll with it :)